graylog2-web-interface/src/components/common/SourceCodeEditor.jsx

```diff
@@ -134,7 +134,7 @@ class SourceCodeEditor extends React.Component {
   render() {
     const { height, width } = this.state;
     const { theme, resizable } = this.props;
-    const validCssWidth = Number.isNaN(width) ? '100%' : width;
+    const validCssWidth = (width === Infinity) ? '100%' : width;
```

Comment:
When resizing the editor, I'm seeing this warning in the console:

```
Warning: `NaN` is an invalid value for the `width` css style property.
```

Reading a bit around, I guess we should use [`lodash.isFinite(width)`](https://lodash.com/docs/4.17.11#isFinite) to check the width. That method checks if the value is a number and not `Infinity`, `-Infinity` or `NaN`.

Reply:
Yep, makes sense. Thx!